src/Map/Map.cs

```diff
@@ -6,20 +6,35 @@
 
 namespace Mapbox.Map
 {
-    using System;
-    using System.Collections.Generic;
-    using Mapbox.Platform;
-    using Mapbox.Utils;
-
-    /// <summary>
-    ///     The Mapbox Map abstraction will take care of fetching and decoding
-    ///     data for a geographic bounding box at a certain zoom level.
-    /// </summary>
-    /// <typeparam name="T">
-    ///     The tile type, currently <see cref="T:Mapbox.Map.Vector"/> or
-    ///     <see cref="T:Mapbox.Map.Raster"/>.
-    /// </typeparam>
-    public sealed class Map<T> : Mapbox.Utils.IObservable<T> where T : Tile, new()
+	using System;
+	using System.Collections.Generic;
+	using Mapbox.Platform;
+	using Mapbox.Utils;
+
+	/// <summary>
+	///     The Mapbox Map abstraction will take care of fetching and decoding
+	///     data for a geographic bounding box at a certain zoom level.
+	/// </summary>
+	/// <typeparam name="T">
+	///     The tile type, currently <see cref="T:Mapbox.Map.Vector"/> or
```

Comment:
You're looking at the wrong repo. Use: https://mapbox.github.io/mapbox-unity-sdk/api/mapbox-sdk-cs/Mapbox.Map.Map-1.html or https://mapbox.github.io/mapbox-sdk-cs/api/Mapbox.Map.Map-1.html.